import "messageContainer.proto";
import "commonEnums.proto";

message predecessorRequestResponse
{
required bytes responder_chord_id = 1; //A 512 bit chord ID
required bytes predecessor_chord_id = 2;  //A 512 bit chord ID

//Add to message container to allow simulated polymorphism
extend messageContainer
{
optional predecessorRequestResponse aPredecessorRequestResponse = 14;
}
}
